web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -1,49 +1,116 @@
-function generateBookmarkList(bookmarks) {
+const uriParams = document.location.search.split('?')[1].split('&');
+const comicName = decodeURI(uriParams[0].split('=')[1]);
+const chapter = uriParams[1].split('=')[1];
+const page = uriParams[2].split('=')[1];
+console.log(`BmcSideBar: comic ${comicName}, chapter=${chapter}, page=${page}`);
+
+function BmcMangaList() {
+    this._node = document.getElementById('manga-list');
+}
+
+BmcMangaList.prototype.onEntryClick = function(comicId) {
+    console.log('clickedOnManga!');
+    window.top.postMessage({comicId}, '*');
+}
+
+BmcMangaList.prototype.generate = function() {
+    var bookmarks = [
+        {label: "naruto",           id: 0},
```

Comment:
A thing I was wondering about: is the `id` supposed to match something in the storage of mangas or is it just used for display?

Reply:
Yes, it is supposed to be an internal unique identifier handled by the storage engine.